### PR TITLE
Replace heavyweight aws-java-sdk with fine-grained plugin

### PR DESCRIPTION
**Pull request by Vlatombe**
Since 1.12.70, the `aws-java-sdk` plugin has been split into multiple fine-grained plugins.

This pull request replaces the heavyweight `aws-java-sdk` (195Mb) with a smaller `aws-java-sdk-minimal` dependency.

- [X] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [X] Ensure that the pull request title represents the desired changelog entry
- [X] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by Vlatombe** [comment]
Ping @alecharp

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.54</version>
+        <version>4.25</version>
```
Comment (alecharp) [inline pom.xml:6]:
```suggestion
        <version>4.27</version>
```

File: pom.xml [hunk 2]
```diff
@@ -86,7 +85,7 @@
         <dependency>
             <groupId>org.jenkins-ci.main</groupId>
             <artifactId>maven-plugin</artifactId>
-            <version>3.4</version>
+            <version>3.12</version>
```
Comment (alecharp) [inline pom.xml:88]:
```suggestion
            <version>3.13</version>
```

**Review by alecharp — CHANGES_REQUESTED** [review]
Just to use the latest version of Maven plugin and of the Parent POM. 
Thanks.

**Review by alecharp — APPROVED** [review]
Thank you @Vlatombe

**Comment by Vlatombe** [comment]
Thanks @alecharp, could you cut a release?

**Comment by alecharp** [comment]
I have test issues locally. Will work on that.

**Pull request merged**